### PR TITLE
Updated the documentation regarding tool behavior when using with public or private CDB endpoint

### PR DESCRIPTION
**Pull request by darshan-patnekar**
Updated the readme file's warning section.

### CODE REVIEW

File: cosmos/sql/tools/chaos/cosmosdb_client_network_chaos_tool/readme.md [hunk 1]
```diff
@@ -2,7 +2,10 @@
...
 - <span style="color:red">**Warning**</span>: 
-    - This tool disrupts the network traffic between the specified target VM(s)/VMSS and the specified Cosmos DB collection/container for the specified account within the specified region but it can potentially affect multiple Cosmos DB accounts in the region. It may also affect other applications running on the specified target VM(s)/VMSS.
+    - This tool disrupts the network traffic between the specified target VM(s)/VMSS and the specified Cosmos DB collection/container for the specified account within the specified region.
+        - If public access is enabled for the Cosmos DB account, the tool can potentially affect multiple Cosmos DB accounts in the region.
```
Comment (RaviTella) [inline cosmos/sql/tools/chaos/cosmosdb_client_network_chaos_tool/readme.md:6]: how about this?	

1. If public access is enabled for the Cosmos DB account, the tool can potentially affect connectivity from the target VM(s)/VMSS to multiple Cosmos DB accounts in the region. 
2. If private endpoint is enabled, then, the tool will affect connectivity from the target VM(s)/VMSS to all the collections in the specified Cosmos DB accounts.

File: cosmos/sql/tools/chaos/cosmosdb_client_network_chaos_tool/readme.md [hunk 2]
```diff
@@ -2,7 +2,10 @@
...
 - <span style="color:red">**Warning**</span>: 
-    - This tool disrupts the network traffic between the specified target VM(s)/VMSS and the specified Cosmos DB collection/container for the specified account within the specified region but it can potentially affect multiple Cosmos DB accounts in the region. It may also affect other applications running on the specified target VM(s)/VMSS.
+    - This tool disrupts the network traffic between the specified target VM(s)/VMSS and the specified Cosmos DB collection/container for the specified account within the specified region.
+        - If public access is enabled for the Cosmos DB account, the tool can potentially affect multiple Cosmos DB accounts in the region. 
+        - If only private endpoint is enabled while the public access is disabled, then, the network disruption will only affect other collections in the account. 
+    - The tool may also disrupt network traffic for other applications running on the specified target VM(s)/VMSS.
```
Comment (RaviTella) [inline cosmos/sql/tools/chaos/cosmosdb_client_network_chaos_tool/readme.md:8]: This is not clear. Not sure if we need this.

File: cosmos/sql/tools/chaos/cosmosdb_client_network_chaos_tool/readme.md (hunk 1 above)
Comment (darshan-patnekar) [reply cosmos/sql/tools/chaos/cosmosdb_client_network_chaos_tool/readme.md:6]: Thanks for framing it correctly! 👍

Comment (darshan-patnekar) [reply cosmos/sql/tools/chaos/cosmosdb_client_network_chaos_tool/readme.md]: Updated

File: cosmos/sql/tools/chaos/cosmosdb_client_network_chaos_tool/readme.md (hunk 2 above)
Comment (darshan-patnekar) [reply cosmos/sql/tools/chaos/cosmosdb_client_network_chaos_tool/readme.md]: Removed.

**Review by RaviTella — CHANGES_REQUESTED** [review]
Please change the PR name to something more relevant.

**Review by RaviTella — APPROVED** [review]
LGTM!

**Pull request merged**